db/migrate/20170519123223_add_request_uuid_to_audits.rb

```diff
@@ -0,0 +1,10 @@
+class AddRequestUuidToAudits < ActiveRecord::Migration
+  def self.up
+    add_column :audits, :request_uuid, :string
```

Comment:
I assume this is now somehow required by 4.0? Do we have to do anything more? Are we sure audits are still getting made?